models/liquidifty/bnb/liquidifty_bnb_trades.sql

```diff
@@ -0,0 +1,212 @@
+            else 'erc20'
+        end as currency_token_standard,
+        '1' as orderType
+    from {{ source('liquidifty_bnb', 'MultipleSell_evt_Buy') }}
+),
+v3 as (
+    select
+        'v3' as version,
+        call_block_time as block_time,
+        get_json_object(nft, '$.id') as token_id,
+        case
+            when get_json_object(nft, '$.assetType') = '0' then 'native'
+            when get_json_object(nft, '$.assetType') = '1' then 'erc20'
+            when get_json_object(nft, '$.assetType') = '2' then 'erc721'
+            when get_json_object(nft, '$.assetType') = '3' then 'erc1155'
+        end as token_standard,
+        case
+            when cardinality(orders) > 1 then 'Bundle Trade'
+            else 'Single Item Trade'
+        end as trade_type,
+        get_json_object(nft, '$.amount') as number_of_items,
+        case
+            when orderType = '0' then 'Swap'
+            when orderType = '1' then 'Buy'
+            when orderType = '2' then 'Sell'
+        end as trade_category,
+        get_json_object(order, '$.signer') as seller,
+        cast(null as varchar(5)) as buyer,
+        get_json_object(currency, '$.amount') as amount_raw,
+        case
+            when get_json_object(currency, '$.assetType') = '0' then '{{ wbnb_address }}'
+            else get_json_object(currency, '$.collection')
+        end as currency_contract,
+        get_json_object(nft, '$.collection') as nft_contract_address,
+        contract_address  as project_contract_address,
+        call_tx_hash as tx_hash,
+        call_block_number as block_number,
+        concat(cast(call_block_number as varchar(5)), '-', call_tx_hash, cast(array_join(call_trace_address, ',') as varchar(5))) as unique_trade_id,
+        case
+            when get_json_object(currency, '$.assetType') = '0' then 'native'
+            when get_json_object(currency, '$.assetType') = '1' then 'erc20'
+            when get_json_object(currency, '$.assetType') = '2' then 'erc721'
+            when get_json_object(currency, '$.assetType') = '3' then 'erc1155'
+        end as currency_token_standard,
+        orderType
+    from (
+        select *,
+            case
+                when orderType = '1' then get_json_object(order, '$.bid[0]')
+                else get_json_object(order, '$.ask[0]')
+            end as nft,
+            case
+                when orderType = '1' then get_json_object(order, '$.ask[0]')
+                else get_json_object(order, '$.bid[0]')
+            end as currency
+        from (
+            select *,
+                get_json_object(order, '$.orderType') as orderType,
+                get_json_object(order, '$.amount') as amount
+            from (
+                select *, explode(orders) as order from {{ source('liquidifty_bnb', 'MarketplaceV3_call_buy') }}
+                union all
+                select *, explode(orders) as order from {{ source('liquidifty_bnb', 'MarketplaceV3_deprecated_call_buy') }}
+            )
```

Comment:
Can we join this info with the `Buy` event, should be possible to match them with the provided `orderHashes`?